.github/workflows/correct_directory.yml

```diff
@@ -0,0 +1,28 @@
+name: directory_linter
+
+ on:
+   pull_request_target:
+     paths:
+       - 'recipes/*.yml'
+       - 'recipes/*.yaml'
+
+ jobs:
+   comment:
+     name: Notify user about wrong dir
+     runs-on: "ubuntu-latest"
+     steps:
+       - uses: actions/checkout@v2
+
+       - name: Comment on PR
+         uses: actions/github-script@v6
+         with:
+           script: |
+             github.rest.issues.createComment({
+               issue_number: context.issue.number,
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               body: 'Hi! Thanks for your contribution to conda-forge.' +
+                     '\nUnfortunately, the recipe was added directly in the _recipes_ folder without its own subfolder.\n' +
+                     'Please move the recipe file (_your package.yaml_) into a folder with the name of the package you want to submit.\n' +
+                     'Thanks!'
```

Comment:
Bit confusing but not sure how to word it better.

Reply:
Maybe use paths? So you could say "recipe was added at `recipes/<feedstock-name>.yaml` and it should be at `recipes/<feedstock-name>/meta.yaml`?

Reply:
Good idea. Let me update that.

Reply:
Do we want `<feedstock-name>` or `your_package`?  `<feedstock-name>` is more consistent but maybe not something everyone knows?